smoothie-stream.py

```diff
@@ -47,19 +49,16 @@ def write_raw_sequence(tn, seq):
 okcnt= 0
 linecnt= 0
 for line in f:
-    tn.write(line)
-    linecnt+=1
-    rep= tn.read_eager()
-    okcnt += rep.count("ok")
-    if verbose: print("SND " + str(linecnt) + ": " + line.strip() + " - " + str(okcnt))
-    if args.log: outlog.write("SND " + str(linecnt) + ": " + line.strip() + " - " + str(okcnt) + "\n" )
-print("Waiting for complete...")
-
-while okcnt < linecnt:
```

Comment:
your seem to have removed the waiting for the ok, which is an error.

Reply:
Oops. Fixed.